### PR TITLE
Migrate to Visual Studio 2022

### PR DESCRIPTION
**Pull request by adrianvmsft**
Migrating to Visual Studio 2022

### CODE REVIEW

File: src/Microsoft.VisualStudio.SlowCheetah.VS/Microsoft.VisualStudio.SlowCheetah.VS.csproj [hunk 1]
```diff
@@ -7,6 +7,25 @@
     <Copyright>Copyright © Microsoft Corporation. All rights reserved.</Copyright>
     <CreateVsixContainer>false</CreateVsixContainer>
   </PropertyGroup>
+  
+  <ItemGroup>
+	  <Content Include="..\Microsoft.VisualStudio.SlowCheetah.SharedFiles\AddTransform_16x.png">
+		  <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```
Comment (chzuluag) [inline src/Microsoft.VisualStudio.SlowCheetah.VS/Microsoft.VisualStudio.SlowCheetah.VS.csproj:13]: nit: Indentation spaces are different in this block

**Review by chzuluag — APPROVED** [review]

File: src/Microsoft.VisualStudio.SlowCheetah.Core/Microsoft.VisualStudio.SlowCheetah.Core.projitems [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    <Compile Include="$(MSBuildThisFileDirectory)Transformer\TransformUtilities.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Transformer\XmlTransformer.cs" />
+    <Compile Update="$(MSBuildThisFileDirectory)Resources\Resources.Designer.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>Resources.resx</DependentUpon>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Update="$(MSBuildThisFileDirectory)Resources\Resources.resx">
```
Comment (adrianvmsft) [inline src/Microsoft.VisualStudio.SlowCheetah.Core/Microsoft.VisualStudio.SlowCheetah.Core.projitems:31]: Are all these needed?

File: src/Microsoft.VisualStudio.SlowCheetah.VS/Microsoft.VisualStudio.SlowCheetah.VS.csproj [hunk 3]
```diff
@@ -7,6 +7,25 @@
     <Copyright>Copyright © Microsoft Corporation. All rights reserved.</Copyright>
     <CreateVsixContainer>false</CreateVsixContainer>
   </PropertyGroup>
+  
+  <ItemGroup>
```
Comment (adrianvmsft) [inline src/Microsoft.VisualStudio.SlowCheetah.VS/Microsoft.VisualStudio.SlowCheetah.VS.csproj:11]: Does adding these files as Content cause them to also become resources in the assembly?

File: src/Microsoft.VisualStudio.SlowCheetah.VS2022/SlowCheetah.vsct [hunk 4]
```diff
@@ -0,0 +1,122 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CommandTable xmlns="http://schemas.microsoft.com/VisualStudio/2005-10-18/CommandTable" xmlns:xs="http://www.w3.org/2001/XMLSchema">
```
Comment (adrianvmsft) [inline src/Microsoft.VisualStudio.SlowCheetah.VS2022/SlowCheetah.vsct:2]: Why not use the file from SharedFiles?

File: src/Microsoft.VisualStudio.SlowCheetah.VSIX2022/source.extension.vsixmanifest [hunk 5]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+    <Metadata>
+        <Identity Id="9eb9f150-fcc9-4db8-9e97-6aef2011017c" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft Corporation" />
```
Comment (adrianvmsft) [inline src/Microsoft.VisualStudio.SlowCheetah.VSIX2022/source.extension.vsixmanifest:4]: Identity needs to be updated

**Pull request closed without merging**